ts/utils/popover.controller.ts

```diff
@@ -25,16 +25,17 @@ module lui.popover {
 			this.body = angular.element(document.getElementsByTagName("body")[0]);
 			this.$scope = $scope;
 			this.clickedOutside = clickedOutside;
+			let that = this;

```

Comment:
petite question : Pourquoi ne pas mettre les deux fonctions en méthode et faire un `.bind(this)` ligne 46 et 53 (et donc garger le this) ?